app/forms/hyrax/etd_form.rb

```diff
@@ -3,10 +3,18 @@
 module Hyrax
   class EtdForm < Hyrax::Forms::WorkForm
     self.model_class = ::Etd
-    self.terms += [:resource_type, :replaces]
+    self.terms += [:date_available, :date_copyright, :date_issued, :date_collected, :date_valid, :date_accepted, :resource_type, :replaces, :degree_level, :degree_name, :degree_field, :contributor_advisor, :contributor_committeemember, :graduation_year, :degree_discipline]
```

Comment:
@vantuyls? Im not entirely sure. I kind of went off assumptions based on the MAP. It's easily fixed if we do need the default metadata for ETD's